modules/installation-guide/partials/proc_mounting-a-secret-or-a-configmap-as-an-environment-variable-into-a-container.adoc

```diff
@@ -0,0 +1,151 @@
+// Module included in the following assemblies:
+//
+// mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-container
+
+[id="mounting-a-secret-or-a-configmap-as-an-environment-variable-into-a-container_{context}"]
+= Mounting a Secret or a ConfigMap as an environment variable into a {prod-short} container
+
+.Prerequisites
+
+* A running instance of {prod}. To install an instance of {prod}, see {link-installing-an-instance}.
+
+.Procedure
+
+. Create a new {platforms-name} Secret or a ConfigMap in the {platforms-name} {orch-namespace} where a {prod-short} is deployed. The labels of the object that is about to be created must match the set of labels:
```

Comment:
Isn't this procedure exactly the same as one in mounting as file?
If so - please consider separating it into dedicated file